### DISCUSSION TITLE
[Done] Customize default `pixel_selection` for mosaic endpoints

### DISCUSSION THREAD

**Discussion by samn** [developmentseed/titiler#492] [Feature Request]
The `pixel_selection` method is defined as a `Query` param, e.g. https://github.com/developmentseed/titiler/blob/cce68aa4c79ac1de43bd9fa5e5ebcf8ab81997ae/src/titiler/mosaic/titiler/mosaic/factory.py#L230-L232 
not as a dependency that can be overridden like other parameters. It would be nice to be able to customize the default mosaic method.

I'm happy to put together a PR adding a new dependency for the mosaic parameters if it sounds ok  to you!

**Comment by vincentsarago** [comment]
> I'm happy to put together a PR adding a new dependency for the mosaic parameters if it sounds ok to you!

🙏, sure that would be lovely!
